ui/tests/pages/secrets/backend/kmip/scopes.js

```diff
@@ -0,0 +1,14 @@
+import { create, clickable, fillable, visitable } from 'ember-cli-page-object';
+import ListView from 'vault/tests/pages/components/list-view';
+
+export default create({
```

Comment:
whew, setting up all of these page objects is a lot of work. props to you for adding them + all these tests!